docs/scos/dev/feature-integration-guides/202204.0/splittable-order-items-feature-integration.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Splittable Order Items feature integration
+last_updated: Nov 22, 2019
+template: feature-integration-guide-template
+originalLink: https://documentation.spryker.com/v3/docs/splittable-order-items-integration
+originalArticleId: ea499c83-900d-4788-bcaf-c7981220b441
+redirect_from:
+  - /v3/docs/splittable-order-items-integration
+  - /v3/docs/en/splittable-order-items-integration
+---
+
+## Install feature core
+
```

Comment:
Add the **Prerequisites** section and fill out the table if there are any other features that must be in place so that your feature could work.

```suggestion

## Prerequisites

To start feature integration, overview and install the necessary features:

| NAME | EXPECTED DIRECTORY | INTEGRATION GUIDE |
| --- | --- | --- |

```